samples/HttpClient/app/application.cpp

```diff
@@ -98,24 +98,28 @@ void setSslFingerprints(HttpRequest* request)
 	// (See: ../Makefile-user.mk )
 	request->setSslOptions(SSL_SERVER_VERIFY_LATER);
 
-	const uint8_t sha1Fingerprint[] = {0xc5, 0xf9, 0xf0, 0x66, 0xc9, 0x0a, 0x21, 0x4a, 0xbc, 0x37,
-									   0xae, 0x6c, 0x48, 0xcc, 0x97, 0xa5, 0xc3, 0x35, 0x16, 0xdc};
+	static const uint8_t sha1Fingerprint[] PROGMEM = {0xc5, 0xf9, 0xf0, 0x66, 0xc9, 0x0a, 0x21, 0x4a, 0xbc, 0x37,
+													  0xae, 0x6c, 0x48, 0xcc, 0x97, 0xa5, 0xc3, 0x35, 0x16, 0xdc};
 
-	const uint8_t publicKeyFingerprint[] = {0x33, 0x47, 0xd1, 0x8a, 0xc8, 0x52, 0xd4, 0xd6, 0xd0, 0xa2, 0xcb,
-											0x3f, 0x4b, 0x54, 0x1f, 0x91, 0x64, 0x94, 0xa0, 0x9c, 0xa1, 0xe2,
-											0xf2, 0x4c, 0x68, 0xae, 0xc5, 0x27, 0x1c, 0x60, 0x83, 0xad};
+	static const uint8_t publicKeyFingerprint[] PROGMEM = {
+		0x33, 0x47, 0xd1, 0x8a, 0xc8, 0x52, 0xd4, 0xd6, 0xd0, 0xa2, 0xcb, 0x3f, 0x4b, 0x54, 0x1f, 0x91,
+		0x64, 0x94, 0xa0, 0x9c, 0xa1, 0xe2, 0xf2, 0x4c, 0x68, 0xae, 0xc5, 0x27, 0x1c, 0x60, 0x83, 0xad};
+
+	SslFingerprints fingerprints;
 
 	/*
 	 * The line below shows how to trust only a certificate in which the public key matches the SHA256 fingerprint.
 	 * When google changes the private key that they use in their certificate the SHA256 fingerprint should not match any longer.
 	 */
-	//	request->pinCertificate(publicKeyFingerprint, eSFT_PkSha256);
+	fingerprints.setSha256_P(publicKeyFingerprint, sizeof(publicKeyFingerprint));
 
 	/*
 	 * The line below shows how to trust only a certificate that matches the SHA1 fingerprint.
 	 * When google changes their certificate the SHA1 fingerprint should not match any longer.
 	 */
-	//	request->->pinCertificate(sha1Fingerprint, eSFT_CertSha1)
+	fingerprints.setSha1_P(sha1Fingerprint, sizeof(sha1Fingerprint));
+
+	// request->pinCertificate(fingerprints);
```

Comment:
Maybe I am missing something but where are the fingerprints attached to the request?

Reply:
In the commented-out line - so behaviour isn't changed. Can un-comment if you prefer, or re-comment `fingerprints.setXXX` lines?

Reply:
Uncomment and change the comments text if needed.

Reply:
OK, I'll also update the fingerprints (they've changed)